flagengine/utils/time.go

```diff
@@ -3,19 +3,19 @@ package utils
 import (
 	"strings"
 	"time"
-)
 
-const iso8601 = "2006-01-02T15:04:05.999999"
+	"github.com/nav-inc/datetime"
```

Comment:
Erk, this library does not look very well supported! It's not been touched in 3 years. I appreciate that it's probably not doing a lot, but have we searched for alternatives, or do we have evidence that this lib is widely used?

Reply:
Switched to https://github.com/araddon/dateparse.